doc/examples/fiber_to_bundle_coherence.py

```diff
@@ -6,9 +6,11 @@
 
 This demo presents the fiber to bundle coherence (FBC) quantitative
 measure of the alignment of each fiber with the surrounding fiber bundles
-[Meesters2016_HBM]_. These measures are useful in “cleaning” the results of
+[Meesters2016]_. These measures are useful in “cleaning” the results of
 tractography algorithms, since low FBCs indicate which fibers are isolated and
-poorly aligned with their neighbors, see Fig. 1.
+poorly aligned with their neighbors, see :ref:`fiber_to_bundle_coherence`.
```

Comment:
I assumed splitting the paragraph would not solve the issue, so I just referenced the figure below in 412e9fb. I'm open to suggestions.